tensorflow_datasets/structured/movielens_utils_test.py

```diff
@@ -0,0 +1,505 @@
+            'movie_title': 'Owl Movie (2018)',
+            'user_id': '1',
+            'user_rating': '3',
+            'timestamp': '997136856',
+            'user_gender': False,
+            'bucketized_user_age': '50',
+            'user_occupation_text': 'retired',
+            'user_occupation_label': 'retired',
+            'user_zip_code': '10278',
+        }),
+        (3, {
+            'movie_id': '2',
+            'movie_genres': ['Crime', 'Sci-Fi', 'Western'],
+            'movie_title': 'Dog Movie (1994)',
+            'user_id': '2',
+            'user_rating': '4',
+            'timestamp': '1188937963',
+            'user_gender': True,
+            'bucketized_user_age': '18',
+            'user_occupation_text': 'college/grad student',
+            'user_occupation_label': 'student',
+            'user_zip_code': '34729',
+        }),
+        (4, {
+            'movie_id': '1',
+            'movie_genres': ['Fantasy', 'Children'],
+            'movie_title': 'Fake Movie (2005)',
+            'user_id': '3',
+            'user_rating': '5',
+            'timestamp': '1083844839',
+            'user_gender': True,
+            'bucketized_user_age': '1',
+            'user_occupation_text': 'scientist',
+            'user_occupation_label': 'scientist',
+            'user_zip_code': '20112',
+        }),
+        (5, {
+            'movie_id': '5',
+            'movie_genres': ['Comedy', 'Romance'],
+            'movie_title': 'Cat Movie (2018)',
+            'user_id': '3',
+            'user_rating': '3',
+            'timestamp': '1042582463',
+            'user_gender': True,
+            'bucketized_user_age': '1',
+            'user_occupation_text': 'scientist',
+            'user_occupation_label': 'scientist',
+            'user_zip_code': '20112',
+        }),
+        (6, {
+            'movie_id': '3',
+            'movie_genres': ['Film-Noir'],
+            'movie_title': 'Non-existent Movie (2003)',
+            'user_id': '4',
+            'user_rating': '2',
+            'timestamp': '803797344',
+            'user_gender': False,
+            'bucketized_user_age': '56',
+            'user_occupation_text': 'other/not specified',
+            'user_occupation_label': 'other/not specified',
+            'user_zip_code': '09583',
+        }),
+        (7, {
+            'movie_id': '4',
+            'movie_genres': ['Comedy', 'Drama', 'Romance'],
+            'movie_title': 'Owl Movie (2018)',
+            'user_id': '4',
+            'user_rating': '1',
+            'timestamp': '1235956810',
+            'user_gender': False,
+            'bucketized_user_age': '56',
+            'user_occupation_text': 'other/not specified',
+            'user_occupation_label': 'other/not specified',
+            'user_zip_code': '09583',
+        }),
+    ]
+    parsed_example_list = list(movies_generator)
+    self.assertListEqual(expected_result, parsed_example_list)
+
```

Comment:
Nit: two empty lines. Surprised the linter doesn't complain.

Reply:
I think the configuration of the lint script in tfds is a bit different. Maybe that's why the linter did not complain. Fixed in 5462c161c468a2b675b154fc9fa169ae6c8af08c.